src/ert/gui/plottery/plots/ensemble.py

```diff
@@ -45,11 +46,13 @@ def plot(
                     plot_context.deactivateDateSupport()
                     plot_context.x_axis = plot_context.INDEX_AXIS
 
+                draw_style = "steps-pre" if is_rate(plot_context.key()) else None
```

Comment:
I tried changing `steps-pre` to something non-sensical to check if it would fail any tests. At least not any tests in `tests/unit_tests/gui` failed. Could you have a check?

Reply:
Apparently there is no "rate" plot tests. Maybe I can add one that will see if draw_style was set?